### ISSUE TITLE
[Eligibility Questionnaire][Accessibility] Consider how to best label the age input

### ISSUE THREAD

**Issue by elenm** [CityOfDetroit/bloom#268]
The age input uses conversational style. Consider whether it should have a screen reader only label of "age", or if we should use "years old" as the label. 
I tried to do some preliminary research on accessible labels for conversational style inputs but didn't find anything definitive.

- [ ] Add a hidden age label for the screenreader

send PR to @elenm

**Comment by willrlin** [comment]
Actually, given that we're going to replace this question entirely in #526, I don't think we need this issue.
